src/controllers/action-queue/index.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export { makeActionProgram } from './ActionProgram'
```

Comment:
We have been on a long campaign to get rid of index files that just re-export things. I feel like this is a hold-over pattern from CommonJS, but in an ES6 named-export world they just make things harder to maintain.

Reply:
I used it as a divider between what is internal API and what's external API to the "controller"